feature/unit tests part 4

yilmazbahadir opened the pull request:
Added unit tests for the following components:
- AccountSelectorPanel
- ContactSelector
- DeleteProfileButton

Commit:
[wallet/desktop] task: unit tests added for AccountSelectorPanel, ContactSelector, DeleteProfileButton components

Jaguar0625 commented on __tests__/components/AccountSelectorPanel.spec.ts:
```diff
@@ -0,0 +1,235 @@
+        const wrapper = getAccountSelectorPanelWrapper(stateChanges);
+
+        // Assert:
+        expect(wrapper.find('div').exists()).toBeTruthy();
```
why truthy vs checking against true ?

yilmazbahadir replied:
updated 👍

Jaguar0625 commented on __tests__/components/ContactSelector.spec.ts:
```diff
@@ -0,0 +1,84 @@
+        // Assert:
+        expect(wrapper.emitted('change')[0][0]).toBe(contactList[0].id);
+        // @ts-ignore - accessing private property
+        expect(component.poptipVisible).toBe(false);
```
might be better to set to true in arrange, to validate action is clearing flag

yilmazbahadir replied:
updated 👍

Commit:
[wallet/desktop] fix: minor review feedback fixes

- setting poptipVisible to true in the arrange portion of the ContactSelector test
- asserting with true instead of truthy in AccountSelectorPanel test
